kafka-rest/src/main/java/io/confluent/kafkarest/response/StreamingResponse.java

```diff
@@ -73,6 +74,7 @@ public abstract class StreamingResponse<T> {
 
   private static final Logger log = LoggerFactory.getLogger(StreamingResponse.class);
   private static final int ONE_SECOND_MS = 1000;
+  private static final int HTTP_SUCCESS_CODE = 200;
```

Comment:
we already have constant for this from either Jetty or Jersey libraries, there is no need to redeclare

Reply:
Thanks for the comment! I wonder how can I access to the constants within Jetty/Jersey? 
<img width="693" alt="image" src="https://github.com/user-attachments/assets/90754d5b-8088-4237-96da-e4c4adc827a1">